src/tools/pkcs15-tool.c

```diff
@@ -1675,6 +1704,34 @@ static void list_info(void)
 			count++;
 		}
 	}
+	printf("\n");
+	for (i = 0; i < SC_MAX_SUPPORTED_ALGORITHMS; i++) {
+		struct sc_supported_algo_info * sa = &p15card->tokeninfo->supported_algos[i];
+
+		if (sa->reference == 0 && sa->reference == 0 && sa->mechanism == 0
+				&& sa->operations == 0 && sa->algo_ref == 0)
+					break;
```

Comment:
clang-format suggested the following change:


```
-		if (sa->reference == 0 && sa->reference == 0 && sa->mechanism == 0
-				&& sa->operations == 0 && sa->algo_ref == 0)
-					break;
+		if (sa->reference == 0 && sa->reference == 0 && sa->mechanism == 0 && sa->operations == 0 && sa->algo_ref == 0)
+			break;
```

<!-- identifier: clang-format -->
<!-- File: src/tools/pkcs15-tool.c -->